### ISSUE TITLE
Zero width space appears as one cell width

### ISSUE THREAD

**Issue by MineRobber9000** [microsoft/terminal#8667]
# Environment

```none
Windows build number: Microsoft Windows NT 10.0.18363.0
Windows Terminal version (if applicable): v1.4.3243.0 (I believe)

Any other software?
WSL (Ubuntu 20.04)
```

# Steps to reproduce

Encounter a zero-width space character. In my case, the easiest way for me to encounter this is to use an IRC client (the zero-width space is commonly used to avoid inadvertant pings in IRC bots).

# Expected behavior

I expect the zero-width space to render as an invisible character, taking up no space in the terminal and not affecting the display of the text.
![WebSSH's terminal display. The zero-width space after the names is not visible.](https://user-images.githubusercontent.com/12676241/103197683-8786be80-48b4-11eb-9b1c-dadbe716aaf8.png)

# Actual behavior

The zero-width space renders as a full space, taking up one monospace cell and affecting the display of the text.
![Windows Terminal display. The zero-width space in the names is clearly visible.](https://user-images.githubusercontent.com/12676241/103197837-c9b00000-48b4-11eb-9a8d-98c9b94205dd.png)

**Comment by skyline75489** [comment]
The Unicode support is not complete yet. I think this is included in #1472 .

**Comment by DHowett** [comment]
Yes, this will be part of /dup #1472. Sorry/thanks!
